src/ipvs/ip_vs_proto_tcp.c

```diff
@@ -97,13 +98,29 @@ static uint32_t tcp_secret;
 /* if tcp header will be modified mbuf_header_pointer() cannot be used. */
 inline struct tcphdr *tcp_hdr(const struct rte_mbuf *mbuf)
 {
-    int ip4hlen = ip4_hdrlen(mbuf);
+    int len;
+    unsigned char version;
+    struct ipv4_hdr *iph = ip4_hdr(mbuf);
```

Comment:
version is not judge as ipv4, should not set iph as v4 header here

Reply:
fixed.